src/MessagingThreadHistory/MessageMetadata.less

```diff
@@ -36,22 +36,6 @@
   align-items: center;
 }
 
-.MessageMetadata--Timestamp--right {
-  color: @neutral_medium_gray;
-  font-size: 0.875rem;
-  .margin--right--xs();
-  .text--line-height-1();
-  white-space: nowrap;
-}
-
-.MessageMetadata--Timestamp--left {
-  color: @neutral_medium_gray;
-  font-size: 0.875rem;
-  .margin--left--xs();
-  .text--line-height-1();
-  white-space: nowrap;
-}
-
```

Comment:
the messaging bubble is now responsible for the `Timestamp` classes/layout/styling